src/components/controller/units/ScheduleUpdateDrawer.vue

```diff
@@ -0,0 +1,230 @@
+      kind="error"
+    >
+      <template #details>
+        {{ fetchError.toString() }}
+      </template>
+    </NeInlineNotification>
+    <div v-else class="flex flex-col gap-6">
+      <NeInlineNotification
+        v-if="sendingError"
+        :description="t(getAxiosErrorMessage(sendingError))"
+        :title="t('controller.units.error_setting_image_update')"
+        kind="error"
+      >
+        <template #details>
+          {{ sendingError.toString() }}
+        </template>
+      </NeInlineNotification>
+      <NeTextInput
+        v-model="versionToUpdate"
+        :disabled="true"
+        :label="t('standalone.update.update_to_version')"
+      />
+      <NeRadioSelection
+        v-model="updateMode"
+        :disabled="sendingSchedule"
+        :label="t('standalone.update.choose_when_to_update')"
+        :options="updateModeOptions"
+      >
+        <template #tooltip>
+          <NeTooltip>
+            <template #content>
+              {{ t('standalone.update.schedule_mode_tooltip') }}
+            </template>
+          </NeTooltip>
+        </template>
+      </NeRadioSelection>
+      <template v-if="updateMode == 'scheduled'">
+        <div>
+          <NeFormItemLabel class="mb-2">{{ t('standalone.update.time_and_date') }}</NeFormItemLabel>
+          <VueDatePicker
+            v-model="scheduledUpdate"
+            :clearable="false"
+            :dark="!theme.isLight"
+            :disabled-dates="(date: Date) => date < new Date()"
+            :time-picker-inline="true"
+            :disabled="sendingSchedule"
+            :readonly="sendingSchedule"
+          />
+        </div>
+      </template>
+      <hr />
+      <div class="flex flex-wrap justify-end gap-4">
+        <NeButton
+          :disabled="sendingSchedule"
+          :readonly="sendingSchedule"
+          kind="tertiary"
+          @click="emit('close')"
+        >
+          {{ t('common.cancel') }}
+        </NeButton>
+        <NeButton
+          :disabled="sendingSchedule"
+          :loading="sendingSchedule"
+          :readonly="sendingSchedule"
+          kind="primary"
+          @click="updateUnit"
+        >
+          <template v-if="updateMode == 'now'">
+            {{ t('standalone.update.update_and_reboot') }}
+          </template>
+          <template v-else>
+            {{ t('standalone.update.schedule') }}
+          </template>
+        </NeButton>
+      </div>
+    </div>
+  </NeSideDrawer>
+</template>
+
+<style>
+/* tailwind theme for vue-datepicker */
+.dp__theme_dark {
+  --dp-background-color: rgb(3 7 18 / var(--tw-bg-opacity));
+  --dp-primary-color: rgb(6 182 212 / var(--tw-bg-opacity));
+  --dp-primary-text-color: rgb(3 7 18 / var(--tw-text-opacity));
+  --dp-border-color-hover: var(--dp-primary-color);
+}
+
+.dp__theme_light {
+  --dp-primary-color: rgb(14 116 144 / var(--tw-bg-opacity));
+  --dp-border-color-hover: var(--dp-primary-color);
+}
+</style>
```

Comment:
This has been already used: is it possible to avoid duplication?

Reply:
Tried again even now, it seems that the external component doesn't like at all tailwind and adding this forces the correct styling for it. I've tried but for now this is all that is possible.

Reply:
Pushed now a change that wraps all CSS in itself, but this is the best that is possible right now.